src/components/BookForm.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const BookForm = (props) => {
+  const { addBookProps } = props;
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (title.trim() && author.trim()) {
+      addBookProps(title, author);
+    }
+  };
+
+  return (
+    <form onSubmit={(e) => handleSubmit(e)}>
+      <input type="text" placeholder="Book title" value={title} onChange={(e) => setTitle(e.target.value)} />
+      <input type="text" placeholder="Book author" value={author} onChange={(e) => setAuthor(e.target.value)} />
+      <button type="submit">ADD BOOK</button>
+    </form>
+  );
```

Comment:
- [OPTIONAL] Great job so far, all good 💯 👍🏻. For now we only need rendering, no functionality like adding and removing books is required, but since you have done that I just wanted to suggest if you could clear your input fields (book title and book author) after adding a book that will enhance your work 😉